src/interfaces/IBPool.sol

```diff
@@ -18,26 +26,85 @@ interface IBPool is IERC20 {
     uint256 tokenAmountOut
   );
 
+  /**
+   * @notice Emitted when a token amount is introduced to the pool
```

Comment:
hmmmmm not, when a join operation is executed or sth like this?

Reply:
Yeah, it sounds weird. My idea was to somewhat explain that the event is emitted when a specific token enters the pool, but I like your idea.
![image](https://github.com/defi-wonderland/balancer-v1-amm/assets/114922365/0b5ffff0-9fdd-4efc-9084-a802c13315fe)